source/blog/2018-02-16-ember-3-1-beta-released.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Ember 3.1 Beta Released
+responsive: true
+author: Matthew Beale
+tags: Releases
+---
+
+Today we're releasing Ember 3.1-beta.1, including releases of Ember.js, Ember
+Data, and Ember CLI.
+
+Traditionally beta releases share a blog post with the
+corresponding stable release, however to keep our messaging clear during the
+transition we've split them up today. See the [Ember 3.0
```

Comment:
I'd remove `today` at the end of the sentence, since the posts have been made on separate days.